HW05B_script.R

```diff
@@ -0,0 +1,69 @@
+#HW05 Part B
+#Author: Hannah Farrell
+#Date: 8/3/2020
+
+#load appropriate packages
+library(tidyverse)
+library(dplyr)
+library(ggplot2)
+library(stringr)
+
+#load in the data
+AUC1 <- read.csv("./CTRP_files/AUC_1.csv", header = TRUE)
+AUC2 <- read.csv("./CTRP_files/AUC_2.csv", header = TRUE)
+cell_line <- read.csv("./CTRP_files/cancer_cell_line_info.csv", header = TRUE)
+compound <- read.csv("./CTRP_files/compound_info.csv", header = TRUE)
+experiment <- read.csv("./CTRP_files/Experiment_info.csv", header = TRUE)
+
+#merge the AUC data
+merged_AUC <- AUC1 %>%
+  full_join(AUC2)
+#add in the 3 other csvs based on matching column names
+full_data_set <- merged_AUC %>%
+  full_join(experiment, by = c("experiment_id" = "expt_id")) %>%
+  full_join(cell_line, by = "master_ccl_id") %>%
+  full_join(compound, by = "master_cpd_id")
+
+#Which cancer type has the lowest AUC values to the compound "vorinostat"?
+lowest_vorinostat <- full_data_set %>%
+  filter(cpd_name == "vorinostat") %>%
+  arrange(area_under_curve) %>%
+  top_n(area_under_curve, n = -20) #pulls out the lowest 20 values for AUC 
+
+ggplot(lowest_vorinostat, aes(x = cancer_type, y = area_under_curve, fill = cancer_type)) +
+  geom_boxplot() +
+  theme(axis.text.x = element_text(angle = 45, vjust = .65))
+#Answer: though not the most elegant/aesthetically pleasing graph, it shows that buy a longshot
+#upper aerodigestive tract cancer has the lowest AUC values with this compound at 4.9941
+
```

Comment:
In this case, I found the use of top_n a lot more useful (there is a lot of compounds). I did not even include a graph to answer this questions because it was not easy to understand, so this is a very good solution.